debian/install

```diff
@@ -1,14 +1,8 @@
-/opt/ros/kinetic/lib/python2.7/dist-packages/nns_ros_bridge/*.py
-/opt/ros/kinetic/lib/python2.7/dist-packages/nns_ros_bridge/msg/*.py
-/opt/ros/kinetic/lib/pkgconfig/nns_ros_bridge.pc
-/opt/ros/kinetic/lib/libnns_ros_bridge.so
-/opt/ros/kinetic/share/gennodejs/ros/nns_ros_bridge/*.js
-/opt/ros/kinetic/share/gennodejs/ros/nns_ros_bridge/msg/*.js
-/opt/ros/kinetic/share/roseus/ros/nns_ros_bridge/msg/*.l
-/opt/ros/kinetic/share/nns_ros_bridge/cmake/*.cmake
-/opt/ros/kinetic/share/nns_ros_bridge/msg/tensors.msg
-/opt/ros/kinetic/share/nns_ros_bridge/package.xml
-/opt/ros/kinetic/share/common-lisp/ros/nns_ros_bridge/msg/*
-/opt/ros/kinetic/include/nns_ros_bridge/tensors.h
-/usr/lib/*/gstreamer-1.0/*
-
+/opt/ros/*/include/nns_ros_bridge/tensors.h
+/opt/ros/*/lib/libnns_ros_bridge.so
+/opt/ros/*/lib/pkgconfig/nns_ros_bridge.pc
+/opt/ros/*/lib/python2.7/dist-packages/nns_ros_bridge/*
+/opt/ros/*/share/common-lisp/ros/nns_ros_bridge/msg/*
+/opt/ros/*/share/gennodejs/ros/nns_ros_bridge/*
+/opt/ros/*/share/nns_ros_bridge/*
+/opt/ros/*/share/roseus/ros/nns_ros_bridge/*
```

Comment:
Where do you install gst plugin? (nnstreamer-ros, which is installed in /usr/lib/gstreamer-1.0 or similar paths)

Reply:
@myungjoo I think I submitted something WiP by mistake. The ```rules``` file also looks bad to me and it seems not the latest version. I will check it at home and re-submit later. Sorry for bothering you. I will label it as DO_NOT_MERGE.